model/riscv_insts_zkn.sail

```diff
@@ -77,6 +77,7 @@ function clause execute (SHA256SUM1(rs1, rd)) = {
  * ----------------------------------------------------------------------
  */
 
+$[name "middle round encrypt"]
```

Comment:
Could you add "AES" to the beginning of the names, as they are displayed in the tables in the ISA specification, sections 32.3.4, 32.3.5. This comment applies to all of the new content.

Also, given that "(RV32)" or "(RV64)" is uniformly present in the tables, please add that to the names as well.